src/applications/login/containers/SignInApp.jsx

```diff
@@ -14,7 +14,7 @@ import downtimeBanners from '../utilities/downtimeBanners';
 const loginHandler = loginType => () => {
   // TODO add separate login tracking for this page
   //   recordEvent({ event: `login-attempted-${loginType}` });
-  login(loginType, 'v1');
+  login(loginType, 'v1', 'cerner');
```

Comment:
does this mean that everyone landing on the standalone login page will be redirected back to cerner?

Reply:
also this is getting changed to "myvahealth" https://github.com/department-of-veterans-affairs/vets-api/pull/4055

Reply:
@ericbuckley I'll update the parameter, as well as refactor it to expect the same query param that the api expects.

ie, `/sign-in?application=myvahealth` will pass along the same query to the sessions endpoint, where other users will just make a normal sessions request.

Reply:
along the lines of avoiding bad side-effects if non-cerner users were to navigate to this page, should the same feature flagging logic that the sign-in modal needs get added here? 

I.e., if someone navigates to this page not from cerner **and** not feature flagged for SSO, should the standalone page also hit the `v0` sessions endpoint?

Reply:
I would think no.  We're not purposely directing users to that page, so while it should work, I don't think we need to guarantee the same experience they would get when logging in via the modal on the home page.

Reply:
Agreed, I don't think we need to feature flag the standalone login page, it only makes sense in SSO scenarios